dotcom-rendering/src/components/HeaderAdSlot.tsx

```diff
@@ -54,12 +49,9 @@ export const HeaderAdSlot = () => (
 			`}
 		/>
 		<Hide when="below" breakpoint="tablet">
-			<div
-				css={[headerAdWrapper, labelStyles]}
-				className="top-banner-ad-container"
-			>
+			<div css={[headerAdWrapper]} className="top-banner-ad-container">
 				<div
-					css={[adSlotContainer, adContainerCollapseStyles]}
+					css={[adSlotContainer, adContainerStyles]}
```

Comment:
Very minor but it might help to distinguish  `adSlotContainer` styles by renaming to `topAboveNavContainerStyles` (or whatever makes sense) and swap the ordering to ensure the top-above-nav styles override anything from the base styles.

```suggestion
					css={[adContainerStyles, topAboveNavContainerStyles]}
```

Reply:
Good spot! Will do